### ISSUE TITLE
House of the Dead 3 EU PC

### ISSUE THREAD

**Issue by IronMaiden2k** [argonlefou/DemulShooter#273]
Argonlefou, I'm at a loss with this game and don't know what l'm doing wrong. I'm trying to play this game with two Sinden Lightguns and the game just hooks both lightguns to player one. I'll try and explain as best as I can what i have so far. My game path is as follows, C:Program Files (x86)\SEGA\THE HOUSE OF THE DEAD3_EU. In this folder, I made a seperate folder for DemulShooter, and I'm using version 10.8.2. I put the arcade mod version v.7.0.0.0 inside the main folder C:Program Files (x86)\SEGA\THE HOUSE OF THE DEAD3_EU ...., I put both players to keyboard, but when I click the launcher with "the arcade stick icon", nothing happens, I "Run As Administrator", nothing happens. I've clicked Demulshooter.exe and Run as Administrator, nothing happens. I configured the DemulShooter GUI for both guns. My script AHK script is as follows:

#SingleInstance force

RUN, "C:\Program Files (x86)\SEGA\THE HOUSE OF THE DEAD3_EU\Hotd3pc_ArcadeMod_v7.0.0.0\Hotd3Arcade_Launcher.exe" 
Sleep, 5000

RUN, "C:\Program Files (x86)\SEGA\THE HOUSE OF THE DEAD3_EU\DemulShooter\DemulShooter.exe" -target=windows -rom=hod3pc -noautoreload -noguns, C:\DemulShooter

; For a KEYBOARD mapping, default keys are 'left control' and 'left shift' for P1
; 'right control' and 'right shift' for P2 and may need to be rebinded to mouse as usual:
LControl::LButton         ;P1 click
LShift::RButton           ;P1 reload
RControl::LButton         ;P2 click
RShift::RButton           ;P2 reload

$Esc::	; Quit Game
	Process,Close,hod3pc.exe
	Run,taskkill /im "hod3pc.exe" /F
	Process, Close, DemulShooter.exe
	Run,taskkill /im "DemulShooter.exe" /F
	ExitApp
return

Please advise what I'm doing wrong. ~Mark

**Comment by argonlefou** [comment]
Hi,
You're doing almost everything wrong : from using a 2 year old DemulShooter binary to your AHK script content (which make me seriously wonder if you even understood what's in it and what DemulShooter is made for)

Nothing personal, but this github issue tracking place is not designed to help people install their setup.

Starting with DemulShooter [wiki](https://github.com/argonlefou/DemulShooter/wiki) would be a good thing

Then in the Sinden discord channel you should be able to get a lot of help for your guns/setup (including from myself if needed) in a way more friendly way than here

**Comment by IronMaiden2k** [comment]
Hi, my apologies if I got you upset. I had one hell of a time getting Night Hunter to work, and after staring at hot key scrips code for hours, I chose a DemulShooter version as per wiki that supposedly worked , changed a few things script wise but managed to get two Sinden guns separated and working. Hod4 and Scarlet Dawn I have working, its this Hod3 that has me stumbled, l did everything from firmware changes to the lightguns to seperate the cameras; I did the hardware ID thing for DemulShooter and ran the GUI so that the two guns are identified and separated. Anyway, I was just curious if it had something to do with the Arcade Mod not recognizing something, thanks at least for the reply.Sent from my iPhoneOn Dec 31, 2024, at 7:47 AM, argonlefou ***@***.***> wrote:﻿
Hi,
You're doing almost everything wrong : from using a 2 year old DemulShooter binary to your AHK script content (which make me seriously wonder if you even understood what's in it and what DemulShooter is made for)
Nothing personal, but this github issue tracking place is not designed to help people install their setup.
Starting with DemulShooter wiki would be a good thing
Then in the Sinden discord channel you should be able to get a lot of help for your guns/setup (including from myself if needed) in a way more friendly way than here

**Comment by argonlefou** [comment]
I'm not upset, I'm just trying to redirect people to the correct place to get the help they're looking for.

In you're search for understanding/installing your games, you're mixing a lot of things that are not doing what you think they are.
For example, the Hod3 arcade mod has nothing to do with sinden guns or inputs, it's just a mod changing the display/gameplay of the game.

Again, you should pass by on the Sinden discord channel : a lot of users mays have overcome some troubles (maybe the same as yours) to setup games, and a lot of them are able to help/explain

**Comment by IronMaiden2k** [comment]
For example, the Hod3 arcade mod has nothing to do with sinden guns or inputs, it's just a mod changing the display/gameplay of the game.

 OMG........ you just solved my entire problem. I changed my script and eliminated the arcade mod direction and *PRESTO* two Sinden Lightguns work seperately. So I'm guessing, instead of me using older versions of DemulShooter as per Wiki, keep my existing DemulShooter folders per game I run, delete their existing contents, and unpack version 13.5 contents into these folders and my games won't know the difference. Sorry for all the trouble, I hate being a f'kin rocket scientist trying to figure this out. Night Hunter kept giving me trouble asking for DemuShooter_v12, so I made a folder called Demulshooter_v12, unpacked v12 in it, and then the game launched with two Sinden Lightguns working independently.  Anyway, Happy New Year to you, I now have a better understanding that scripting is about combining game launch EXE to Lightgun EXE, thanks again \m/
